exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,12 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+  return (mark >= 80) ? "A" :
+         (mark > 60) ? "B" :
+         (mark >= 50) ? "C" :
+         "F";
+}
 
```

Comment:
I'm not sure but when we separate the intervals as like this, does it cause a potential error? like writing 
(mark > 60) instead of (mark < 80 && mark > 60)

Reply:
I was wondering the same question too. I asked our "guardian-study-group" mentors last week. they said it will "return" directly if any condition is "true". By the way, thanks to my Tech Buddy (Matt), I just learned about "ternary operators" last week. I have to do much more practice🤭

Reply:
Hi Musa, a ternary operator is great for when you have a simple true/false boolean conditional. In this instance there are multiple boolean conditionals, whilst you can use a ternary operator here and it works a cleaner solution would be to use a switch statement. Here are the docs: https://www.w3schools.com/js/js_switch.asp